Description: include stdio.h to fix failure due to undeclared NULL

Author: Gianfranco Costamagna <locutusofborg@debian.org>
Last-Update: 2018-05-16

--- boinc-app-seti-8.00~svn3725.orig/client/vector/sighandler.h
+++ boinc-app-seti-8.00~svn3725/client/vector/sighandler.h
@@ -47,6 +47,10 @@
 #include <signal.h>
 #include <setjmp.h>
 
+#if defined(HAVE_SIGACTION)
+#include <stdio.h>
+#endif
+
 #ifndef MAXSIG
 
 #ifdef NSIG
